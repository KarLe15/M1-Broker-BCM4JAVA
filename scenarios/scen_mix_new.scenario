## operation, component, scenario, uri
CREATE,subscriber,scen_mix_new,subscriber_1
CREATE,subscriber,scen_mix_new,subscriber_2
CREATE,subscriber,scen_mix_new,subscriber_3
CREATE,publisher,scen_mix_new,publisher_1
CREATE,publisher,scen_mix_new,publisher_2
CREATE,publisher,scen_mix_new,publisher_3
