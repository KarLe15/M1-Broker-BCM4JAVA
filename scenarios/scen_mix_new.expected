{
"subscriber_1" : 963,
"subscriber_2" : 963,
"subscriber_3" : 963
}
